app/Db/leyboldim540_ch14.template

```diff
@@ -0,0 +1,397 @@
+#
+# Leybold IM540
+#
+#  DEV
+#  	EPICS prefix
+#  PORT
+#  	ASYN PORT_NAME
+#  CH
+#   CHANNEL NUMBER
+#
+#
+#record(mbbi, "$(DEV):GET_OFC_CH$(CH)")
```

Comment:
Question: I notice many of these records are commented out. Is this scope for later or can they be removed?

Reply:
Yes, that's true there are many commands commented out. Some got commented out because are related only to controller's CH3 and CH4 which, based on Jing's comment will not be used (CH3 and CH4 supports different sensor then CH1 and CH2). Some got commented out because were not needed and because I was getting random timeouts on PVs and that's why I scaled down the number of commands a bit.